CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 # History
 
+## 3.2.0
+- Widget:
+    - The methods `UploadcareWidget.selectFile()` and `Fragment.startActivityForResult` have been deprecated.
```

Comment:
Is it BREAKING or breaking? :-)

Reply:
It's not breaking, because they were deprecated, not removed or replaced, clients can keep using them if they don't want to migrate to the new mechanism